src/main/kotlin/org/jetbrains/research/ml/dataset/anonymizer/transformations/anonymization/java/JavaTypes.kt

```diff
@@ -0,0 +1,25 @@
+package org.jetbrains.research.ml.dataset.anonymizer.transformations.anonymization.java
+
+import com.intellij.psi.*
+import org.jetbrains.research.ml.dataset.anonymizer.transformations.anonymization.JvmTypes
+
+object JavaTypes : JvmTypes() {
+    override fun isClass(element: PsiElement?): Boolean = !isInterface(element) && element is PsiClass
+    override fun isFunction(element: PsiElement?): Boolean = element is PsiMethod
+    override fun isStaticFunction(element: PsiElement?): Boolean = element is PsiMethod && element.isStatic()
+    override fun isNonStaticFunction(element: PsiElement?): Boolean = element is PsiMethod && !element.isStatic()
```

Comment:
Can we use `isFunction` instead of `element is PsiMethod`?

Reply:
Actually, it ruins the magic of Kotlin smart cast, but sure, I'll fix it